examples/common.cpp

```diff
@@ -77,6 +77,7 @@ std::string process_escapes(const char* input) {
             if (input[i] == '\\' && i + 1 < input_len) {
                 switch (input[++i]) {
                     case 'n':  output.push_back('\n'); break;
+                    case 'r':  output.push_back('\r'); break;
                     case 't':  output.push_back('\t'); break;
                     case '\'': output.push_back('\''); break;
                     case '\"': output.push_back('\"'); break;
```

Comment:
wait, is this even right? why do we escape `"` in a char literal

Reply:
Yeah, it's only testing the char literal when the previous character was `\`, so this turns `\"` into `"`.

I just tried to copy the most common escapes used by programs like `sed`.

Reply:
Oh, I see what you're saying.

Reply:
No, I ment, why `'\"'` instead of just `'"'`

Reply:
Oh, I misunderstood your question. Yeah, we don't need to escape quote (`\"`) inside the single quote. I just did that to be consistent and make things line up better.